doc/source/basics.rst

```diff
@@ -541,7 +555,7 @@ will exclude NAs on Series input by default:
 .. ipython:: python
 
    np.mean(df['one'])
-   np.mean(df['one'].values)
+   np.mean(df['one'].array)
```

Comment:
```suggestion
   np.mean(df['one'].to_numpy())
```

(I think the goal here is to show the difference with a numpy array, so I find using `to_numpy` more appropriate)